front/app/components/SignUpIn/SignUp/stepUtils.test.ts

```diff
@@ -0,0 +1,286 @@
+import { getDefaultSteps, getActiveStep, getEnabledSteps } from './stepUtils';
```

Comment:
These unit tests might need to be refactored. There is a lot of module-specific code in here that should probably live in the corresponding module. The annoying thing is that the crucial thing this code tests is how different modules work together. I am not sure about the best way to structure unit tests in this case.

Reply:
Will do this later